marshaller_test.go

```diff
@@ -286,6 +288,123 @@ var _ = Describe("Marshaller", func() {
 			})
 		})
 
+		Context("when all clauses are signed integer data types", func() {
```

Comment:
Can we also add test for mixed types? Query criteria that has some string, some integer, some time fields.

Reply:
Will do